manager/controllers/package_systems.go

```diff
@@ -24,7 +24,8 @@ type PackageSystemsResponse struct {
 func packageSystemsQuery(acc int, pkgName string) *gorm.DB {
 	return database.Db.
 		Table("system_platform").
-		Joins("inner join system_package spkg on spkg.system_id = system_platform.id").
+		// nolint: lll
+		Joins("inner join system_package spkg on spkg.system_id = system_platform.id and system_platform.stale = false").
```

Comment:
and spkg.rh_account_id = system_platform.rh_account_id

Reply:
Resolved